test/com/google/enterprise/adaptor/filenet/TestObjectFactory.java

```diff
@@ -75,7 +75,7 @@ public static AccessPermissionList getPermissions(
 
   // TODO(tdnguyen): Combine this method with similar methods in
   // PermissionsTest.
-  private static AccessPermission newPermission(String granteeName,
+  public static AccessPermission newPermission(String granteeName,
```

Comment:
I've been troubled by exposing this method for years, for some reason, which I can't put my finger on. Let's at least implement the TODO (PermissionsTest.addAces is basically `perms.add(newPermission(...))`), and also replace your use of `AccessRight.READ_AS_INT | AccessRight.VIEW_CONTENT_AS_INT` and others in this class with `Permissions.VIEW_ACCESS_RIGHTS`. I think there are tests that vary the permissions, and those need not use the constant, since the permutations are interesting to see, but in standalone cases where we are referring specifically to the minimum required permissions, I think we should use the constant more.

I'm happy for this to be a followup CL if you prefer, though I think it ties in with making this method public.